### DISCUSSION TITLE
Enabling displays for the esp32 build

### DISCUSSION THREAD

**Discussion by Nimo789** [BitMaker-hub/NerdMiner_v2#497] [Q&A]
I stumbled across the following issue: I am trying to connect a small 1.3" OLED IIC Display to my ESP32 WROOM NodeMCU in order to display the mining stats. As far as I found out by now, the SDA pin of the display should connect to pin G21 of the ESP, while the SCL pin should go to pin G22. GND is connected to Gnd and VCC either to 3V3 or V5. However, I have no clue what I have to modify in the code or the PlatformIO file... I tried to set "-D LED_DISPLAY=1" in the Platformio file, but this only leads to critical errors during the build proceds and the file can not be built at all... what am I missing here? Lateron I Plan to connect a 2.8" display to another esp32, but this has even more Pins than the smaller one.

**Comment by paelzer** [comment]
The original Nerdminer V2 repo doesn't support I2C displays afaik. Anyway it's easy to achieve...

I set up something like this a while ago, maybe it's helpful. Have a look in this repo:
https://github.com/paelzer/NerdMiner_v2_paelzer/tree/ESP32_devkit_C_SSD1306

And find some information related here:
https://github.com/paelzer/NerdMiner_v2_paelzer/tree/ESP32_devkit_C_SSD1306/images/ESP32_SSD1306

**Comment by G5OYY-uk** [comment]
did you manage to do?? 
trying to add display to a couple of esp32 s3 boards but just keep breaking the firmware lol
